test/fixtures/git-credentials-1.txt

```diff
@@ -0,0 +1 @@
+ABC
```

Comment:
I would like to put here examples of correct files that we want to support. Personally I'm not using git-credentials but `SSH`. @jesusprubio I'd be thankful for a few example from you :)

Reply:
cat .git-credentials                                                                                                                     ✔  
https://jesusprubio:MY_TOKEN@github.com
https://jesusper%40workdomain.com:MY_ENTERPRISE_TOKEN@github.workdomain.com

Reply:
Ok, so seems like it's possible to have multiple tokens in single `git-config` file. Which one should I choose ? :thinking: Or maybe I need to extend `readGithubToken` method API ?

Reply:
I think for now we can get the first one, it seems to be always the public GitHub one. Later I think it would be nice to ask him with a selector, showing the email or something.